packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -235,8 +235,13 @@ const prompts = () => {
         },
         {
             name: 'siteId',
-            message: "What is your site's ID (examples: RefArch, SiteGenesis) in Business Manager?",
-            validate: validSiteId
+            message:
+                "What is your site's ID (examples: RefArchGlobal, SiteGenesis) in Business Manager?",
+            validate: validSiteId,
+            default: () => {
+                const defaults = testProjectAnswers()
+                return defaults[['commerce-api']].siteId
```

Comment:
Just to double check.. does it need to be double square brackets?

Reply:
Thanks, Vincent, good eye. Kev also saw the same thing.